server/api/content.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const router = express.Router();
+
+router.get("/:category", (req, res) => {
+  const { category } = req.params;
+  res.send(require(`../json/${category}.json`).slice(0, 4));
+});
+
+router.get("/:category/:page", (req, res) => {
+  const { category } = req.params;
+  const page = parseInt(req.params.page);
+  const content = require(`../json/${category}.json`);
+  res.send(content.slice((page - 1) * 12, page * 12));
```

Comment:
여기서 12는 상수로 분리해주면 어떨까요?

Reply:
```suggestion
  const start = (page - 1) * 12;
  const end = start + 12;
  res.send(content.slice(start, end));
```
이런식으로 표현할 수 있을 것 같아요!